lib/src/migrator.dart

```diff
@@ -68,9 +68,11 @@ abstract class Migrator extends Command<Map<Uri, String>> {
   Map<Uri, String> run() {
     var allMigrated = <Uri, String>{};
     var importer = FilesystemImporter('.');
-    var importCache = ImportCache(
-        importers: [NodeModulesImporter()],
-        loadPaths: globalResults!['load-path']);
+    var importCache = ImportCache(importers: [
+      NodeModulesImporter(),
+      if ((globalResults!['pkg-importer'] as List<String>).contains('node'))
```

Comment:
I think it would communicate the likely extensions of this behavior better to make this a for loop.